src/hotspot/os/linux/os_linux.cpp

```diff
@@ -4646,6 +4656,15 @@ jint os::init_2(void) {
     FLAG_SET_DEFAULT(UseCodeCacheFlushing, false);
   }
 
+  // Override the timer slack value if needed. The adjustment for the main
+  // thread would inherit the setting the children threads, which would be
+  // most threads in JDK/JVM.
```

Comment:
s/would inherit the setting the children threads/will establish the setting for child threads/

Reply:
Done, thanks.